### PR TITLE
riff driver: use kafka directly vs. via riff http gateway

### PR DESCRIPTION
**Pull request by imikushin**
FunctionExecution:
- .ID -> .FunctionID
- rm .Name
- add .RunID

riff driver:
- use kafka
- Close()
- configure to use kafka broker

e2e tests:
- update e2e/tests-riff
- wait until the run is READY when checking for `.output`

### CODE REVIEW

File: charts/dispatch/charts/function-manager/templates/config-map.yaml [hunk 1]
```diff
@@ -21,7 +22,7 @@ data:
           "funcNamespace": "{{ .Values.faas.openfaas.namespace }}"
         },
         "riff": {
-          "gateway": "{{ .Values.faas.riff.gateway }}",
+          "kafkaBrokers": ["{{ index .Values.faas.riff.kafkaBrokers 0 }}"],
```
Comment (kars7e) [inline charts/dispatch/charts/function-manager/templates/config-map.yaml]: helm uses [sprig](https://godoc.org/github.com/Masterminds/sprig), so this should work:
```
"kafkaBrokers": ["{{ join "," .Values.faas.riff.kafkaBrokers }}"],
```

File: pkg/utils/try-close.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
+///////////////////////////////////////////////////////////////////////
```
Comment (kars7e) [inline pkg/utils/try-close.go]: NIT: should use _ instead of - in file name (but TBH, `close.go` is a perfectly fine name for this as well :) )

Comment (imikushin) [reply pkg/utils/try-close.go]: yep, `close.go` works

File: charts/dispatch/charts/function-manager/templates/config-map.yaml (hunk 1 above)
Comment (imikushin) [reply charts/dispatch/charts/function-manager/templates/config-map.yaml]: Thanks! Fixed

**Pull request merged**